docs/docs/choose-provider/index.md

```diff
@@ -0,0 +1,135 @@
+
+* **Virtual Private Cloud (VPC)**
+AWS VPC is a Virtual Network! It allows you to create an complete logical network infrastructure for your VMs. This opens up a world of possibilities for using multiple VMs together. If you don't require any special networking, Koding handles this automatically. However, if you want to create a custom network environment, you can configure it simply within your Stack.
+
+ * ***[AWS Documentation](https://aws.amazon.com/vpc/)***
+ * ***[Using IAM in your Stack](/docs/create-an-aws-vpcr)***
+ * ***[Stack Reference](/docs/awc_vpc)***
+
+***
+
+## Vagrant <a name="vagrant"></a>
+
+Vagrant allows you to use your own machine to host your Koding VMs. You will install it along-side VirtualBox, so Vagrant can interface you with all of its features. 
+
+***[Creating a Stack with Vagrant](/docs/vagrant/creating-a-stack-with-vagrant)***
+***[Vagrant Website](https://www.vagrantup.com/)***
+***[Vagrant Stack Reference](/docs/vagrant/vagrant-stack-reference)***
+
+> Since (VirtualBox-based) Vagrant VMs do not exist in the Cloud, backing up your data is all the more important. If the machine that hosts your VMs dies, you'll lose those VMs and everything on them. Your Koding Stack will not be affected, so getting started with a new VM on a new machine will be quick and simple! 
+
+***
+
+## DigitalOcean <a name="digital-ocean"></a>
+
+***[Getting Started with DigitalOcean](/docs/getting-started-digital-ocean)***
+***[DigitalOcean Website](https://www.digitalocean.com/)***
+***[DigitalOcean Stack Reference](https://www.terraform.io/docs/providers/do/index.html)***
+
+DigitalOcean is a very popular option among developers an teams who don't need all of the features of AWS or GCE and also don't need all of the headache. Its feature-set is much smaller than the others, but those it does offer are the features most used by small teams. Documentation is limited (though this is countered by a strong active community), but since they are such a straightforward service, there isn't much required. DigitalOcean's pricing is among the best, in large-part due to their bare-bones approach. 
+
+> Using DigitalOcean as a provider will require you to [Sign Up](https://cloud.digitalocean.com/registrations/new) for the service. Currently, this requires a credit card and for you to pre-load your account with at least $5 (the amount needed for the lowest-tier droplet for a month.) Using DigitalOcean in your Stack will require you to create and copy an Access Token, [here](https://cloud.digitalocean.com/settings/api/tokens). This is gone over in greater detail in our [Getting Started with DigitalOcean](/docs/getting-started-digital-ocean) guide.
+
+* **Floating IP**
+A Floating IP is a static public IP address that can dynamically map to any VM. This can be used to mask failures by re-routing domains to different instances without needing to change the DNS records. Configuring this is very simple (pick one of your droplets and click a button).
```

Comment:
If we can configure floating IPs in Koding, then let's not mention selecting a droplet and clicking a button.